### ISSUE TITLE
example1 correct pumping power?

### ISSUE THREAD

**Issue by softwareengineerprogrammer** [NREL/GEOPHIRES-X#156]
Per @kfbeckers via email:

Are we sure GEOPHIRES example 1 is still calculated correctly?

We are producing 50 MW of heat but only get 1 MW of net power because the pumping power is so high. I was wondering why that is? Originally, example 1 had lower pumping power I believe but I know some updates have recently been implemented that may have changed the pumping power? I increased the wellbore diameter and the injectivity/productivity index but they had limited impact.

![image](https://github.com/NREL/GEOPHIRES-X/assets/4056124/0ebcd2be-2d38-4128-b938-fc46fca25493)

**Comment by softwareengineerprogrammer** [comment]
@kfbeckers The pumping power increased from 0.21MW to 4.37MW when we updated calculations to account for pressure, see:

https://github.com/NREL/GEOPHIRES-X/blob/48f15b4705be0a03532d061a90a8959eb0c33f8a/src/geophires_x/WellBores.py#L358-L368

Do these calculations look correct to you?

**Comment by kfbeckers** [comment]
Yes, it appears the vapor pressure equation of water is not correct. Right now, we have CP.PropsSI('P','T','P') which will just return the 'pressure' we provide to the function. By definition, the vapor pressure is the pressure at which the fluid starts boiling for a given temperature, and hence solely depends on temperature. The equation we have in the "else" statement ((quantity(CP.PropsSI('P', 'T', celsius_to_kelvin(Twater_degC), 'Q', 0, 'Water'), 'Pa')
                    .to('kPa').magnitude))
is the correct one. Maybe using that one should should fix the pumping power?

![image](https://github.com/NREL/GEOPHIRES-X/assets/38221976/e9083662-9e35-4d4e-a6dc-7f44a5b2d89d)

**Comment by softwareengineerprogrammer** [comment]
@kfbeckers I staged a PR for my fork with the change you suggest - can you take a look at the effect on the example outputs and see if it looks right? View at https://github.com/softwareengineerprogrammer/GEOPHIRES-X/pull/19/files#diff-3649ae9ab87dcc5ad182ea6a25b745e05b9cf05f17b8d7ad468d4b0ab525e2b7 (you will probably need to press "Load Diff" for each of the `.out` files to see the difference)
